### ISSUE TITLE
Some pattern scroll ingredients don't match effects

### ISSUE THREAD

**Issue by vlad1492** [chaosvolt/cdda-arcana-mod#109]
Really enjoy the mod, thank you for your work. 

Noticed an issue with some Pattern scrolls calling for very different inputs. 

Examples: 
Frost Armor calls for divine scroll of darkness
Summon Centipedes calls for scroll of sundering
Elemental Burst calls for scroll of regrowth
Sundering Quake calls for scroll of command

Not sure when this changed, I seem to recall they lined up the last time I played a scholarly type. 
Not even 100% sure these patterns scrolls come from your mod, if not then apologies. 

- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.D-10681-gab89765 [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Magiclysm [magiclysm],
    More Survival Tools [More_Survival_Tools],
    Makeshift Items Mod [makeshift],
    Icecoon's Arsenal [ew_pack],
    Medieval and Historic Classes and Shields [Medieval_Stuff],
    PKs Rebalancing [PK_REBALANCE],
    More Locations [more_locations],
    Fuji's More Buildings [FujiStruct],
    Folding Parts pack [deoxymod],
    Vehicle Additions Pack [blazemod],
    Arcana and Magic Items [Arcana],
    Bionic Slots [cbm_slots],
    Fuji's Military Profession Pack [fuji_mpp],
    Manual Bionic Installation [manualbionicinstall],
    Safe Autodoc [safeautodoc],
    Sleep Deprivation [sleepdeprivation],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills]
]

**Comment by chaosvolt** [comment]
These were new spells added to flesh some ideas I had out. Before I added that, the pattern scrolls available in The Six Pillars where basically just picking 2 spells from each faction. Of those, Overgrowth matched one of the effects of the scroll of regrowth, while Serpentine Shield is perhaps a more beneficial version of the side effects from scrolls of discord.

The others however really didn't fit the effects of the other scrolls: Might, Clarity of Blood, Flame Armor, and Acid Spray. The main thematic link behind both the old spell recipes, and the new ones I added, isn't in what effects they share. The intended link is what they imply about the entities The Six Pillars is supposed to describe, each of which is mentioned in the description of the six divine scrolls.

None of the new pattern scrolls really match up with the effects of any of the divine scrolls used to make them, though Eclipse does retain the general theme of "invite things that don't belong here" that the scroll of discord has, but it also shares some effects with scrolls of darkness.

Consecrate is a reasonably natural fit for the scroll of light thematically, since it makes sense given the holy theming despite the different effect. All the others are a more noticeable divergence in theme, which I probably could flesh out better to explain why the effects are so different.

That said, I was thinking of expanding on the recipes for each pattern scroll, since making them from just a divine scroll and no other thematically-appropriate items (like monsterparts with more closely-related effects) doesn't help much with explaining why the effects change.

**Comment by chaosvolt** [comment]
I went with the idea I mentioned at the end, to better provide a practical link to the divine scroll and the pattern scroll that can be made from them. My idea was requiring, in addition to the thematically-intended divine scroll, one of two magic seals, the options for each generally being closer in general effect, at last enough to make it a bit more intuitive.

* For Consecrate, this requires adding either Healing or Ward Against Evil, with the implication being a change to affect the environment instead of a living being.
* Frost Armor requires adding either Flame Armor or Lightning Ward. In this case the use of a different elemental protective spell to create a different one is the intended link.
* Elemental Burst uses either Shockstorm or Conjure Acid, which are of course the two elemental offensive spells you can craft magic seals for (in addition to being two of the possible outcomes of the spell).
* Eclipse uses Serpentine Shadows or Insight, both of which are fairly thematically relevant to Eclipse (and the former is in the general category of "otherworldly monsters slipping through the Veil" and such, giving it a link in terms of effect).

There's two where I still couldn't really make the connection as intuitive as I would've liked:
* Sundering Quake requires adding either Might or Overgrowth. The latter makes sense as it's another destructive terrain-altering effect, but the former is a bit of a "none of the magic seals really involve earth and stone" thing.
* Needing Clarity of Blood for Summon Centipedes is reasonably logical thematically, given the whole "summoning vermin" thing. Having Confuse Monsters only really makes sense if one notes that the magic seal for it is made from a barbed stinger. @.@

Sorry for the confusion, and hope that makes it a bit less nonsensical.

**Comment by vlad1492** [comment]
I haven't tried the updated version yet, sounds like a winner to me. I like it!

Can't help sharing an enthusiasm about the mod, apologies if this isn't the venue. 
tl;dr at the bottom. 

My latest adventure worked so well.

My survivor had found The Six Pillars a while back but didn't know what to do with it. made a copy of the notes to pack around just in case, left the original back at base. 

While exploring in his electric APC he came across a single-level lab. In the containment was a Shoggoth. He got to practice with the nifty laser rifle stolen from the last lab by firing right through the glass. When that got old he contined to explore. Down in one corner of the facility were two tears in reality and a whole lot of otherworldy badness kept coming through.  

The encounter didn't go well, by which I mean the very best kind of terrified barely-survived escape.

All the creepies were after him, with the flying chainsaws being worst.  After exhausting his UPS with laser-fire he switched to slug-thrower rifle, then the spear.  Injuries were ramping up, pain was becoming a problem.  Plus slime and shakes and more.  Retreated through an area he hadn't scouted yet - oops. Bunch of flaming eyes. Blinded and terrified he managed to fumble his way above ground, dropping the heaviest loot just to get away.  Including the laser rifle.

The APC was parked outside the lab, safe place to heal up and consider options. While patching his hurts and mending his gear he started coughing badly.  Nasty grey vomit-type coughing. Ah, hell. Bad radiation too.

Race back to base camp, slamming over a huge boulder and ripping out part of the frame. Got back to find I had *almost* enough hardware to put together a home-brew pharmacy.  Had to go back out to scrounge some glassware and rubber hoses.  In the nick of time cooked up some antifungal treatment  - torso was down to two vertical dots, no bars.

It took a long while for my slow healer to recuperate, time to study up that book and put it to work. He had grabbed some materials from the lab and had my stash of supplies.  Water signs to speed healing, and then eventually learned of Consecration.

The set of Japanese armor was lost when the boulder tore out the APC frame- the gear had been stored in that section. Time to gear up and take the lab back in the name of all that is right and holy.   Built some armor, loaded more ammo, found some resistance to radiation. 

Make my way back, fight through the teeming eldritch hordes, curse the amoeba that ate my laser rifle, get right next to the rift and summon the pattern of Consecration. Bam!


Nothing happened.


tl;dr: What does consecrate do?  Is there a spoiler sheet for the mod ?

**Comment by vlad1492** [comment]
Ah, found the details in mapgen_update.json

**Comment by chaosvolt** [comment]
No worries. Yeah, its effects are kinda overly narrow. If we ever get the ability to filter out valid targets by monster species, I want to one day make it specifically damage and/or debuff monsters of species `NETHER`, maybe some mutants too.

As for the ter_transform, I really oughta move them to a separate file, instead of being lumped in with mapgen updates. Hell, I oughta reorganize a lot of stuff, and I'm hoping to have free time to do so when I get prepared for setting aside a 0.E release.
